package.json

```diff
@@ -28,8 +28,8 @@
     "clean:r11s": "cd server/routerlicious && npm run clean",
     "postinstall": "npm run postinstall:lerna",
     "layer-check": "fluid-layer-check --info tools/build-tools/data/layerInfo.json",
-    "lint": "lerna run lint --no-sort --stream",
-    "lint:fix": "lerna run lint:fix --no-sort --stream",
+    "lint": "lerna run lint --no-sort --stream -- -- -- --color",
+    "lint:fix": "lerna run lint:fix --no-sort --stream -- -- -- --color",
```

Comment:
hahaha, why so many `--`? Isn't just one enough? Ohhh, this is because of all the npm script nesting, right?

Reply:
Yeah my earlier failed attempt used 3x `--` which was... insufficient